src/Support/Eventable.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace KirschbaumDevelopment\NovaMail\Support;
+
+use Illuminate\Support\Str;
+use Symfony\Component\Finder\Finder;
+use Illuminate\Database\Eloquent\Model;
+use KirschbaumDevelopment\NovaMail\Traits\Mailable;
+
+class Eventable
+{
+    private static $defaultPaths = [
+        '',
+        'Models',
+    ];
+
+    /**
+     * Eloquent models that events should be listened on.
+     *
+     * @var array
+     */
+    private static $models = [];
+
+    /**
+     * Register all of the resource classes in the given directory.
+     *
+     * @return void
+     */
+    public static function discoverModels()
+    {
+        $namespace = app()->getNamespace();
+
+        $directories = collect(self::$defaultPaths)
+            ->merge(config('nova_mail.model_paths'))
+            ->transform(function($model) {
+                return app_path($model);
```

Comment:
@brandonferens Does this mean we would be restricted to the app directory - even for custom paths? Seems like `base_path` would be better for maximum flexibility.

Reply:
@adammparker This wasn't an easy change, however, I got it done and I think it is a lot better than than assuming the `App` path and namespace.